pkg/settings/manager.go

```diff
@@ -0,0 +1,122 @@
+package settings
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"time"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+// Settings represents user settings.
+type Settings struct {
+    // OpenaiKey is an OpenAI API Key to use with Assistant for GraphQL queries.
+	OpenaiKey string `json:"openai_key"`
```

Comment:
It looks like this file isn’t properly formatted according to Go’s standards. 

Go has an ‘official’ formatter called `go fmt` that helps with this. You can configure your editor (WebStorm?) to automatically run `go fmt` on each Go file when you save it.